packages/exhibitor/build

```diff
@@ -9,6 +9,8 @@ rm -rf "$PKG_PATH/usr/zookeeper/"{src,docs,contrib}
 
 mkdir -p "$PKG_PATH/bin"
 ln -s "$PKG_PATH/usr/zookeeper/bin/zkCli.sh" "$PKG_PATH/bin/zkCli.sh"
+# zkCli.sh expects zkEnv.sh to be in the same directory.
+ln -s "$PKG_PATH/usr/zookeeper/bin/zkEnv.sh" "$PKG_PATH/bin/zkEnv.sh"
```

Comment:
When people used it they ran it from the package directory, e.g. `/opt/mesosphere/packages/exhibitor--9b89c1f1654d9f2addfb8dfaa138053d6c6a5eb7/usr/zookeeper/bin/zkCli.sh`.